src/app/restaurant/[id]/page.tsx

```diff
@@ -39,17 +39,22 @@ const RestaurantsPage = (props: { params: { id: string } }) => {
           {data.reviews.map((review) => (
             <div
               key={review.createdBy}
-              className="flex flex-row justify-between p-2 even:bg-stone-400 dark:even:bg-stone-600"
+              className="mb-4 flex flex-col rounded-lg bg-stone-100 p-4 dark:bg-stone-800"
             >
-              <p>Rating: {review.rating}</p>
-              <p className="flex flex-col text-xs text-stone-700 dark:text-stone-300">
-                {new Date(review.createdAt).toLocaleDateString()}
-                {user.sub === review.createdBy ? (
-                  <span className="text-red-700 dark:text-red-300">
-                    Your review
-                  </span>
-                ) : null}
+              <div className="flex items-center justify-between">
+                <p className="font-semibold">RATING: {review.rating}</p>
+                <p className="text-xs text-stone-700 dark:text-stone-300">
+                  {new Date(review.createdAt).toLocaleDateString()}
+                </p>
+              </div>
+              <p className="mt-2 text-stone-900 dark:text-stone-100">
+                {review.comment}
               </p>
```

Comment:
tän vois viel ehkä miettii et ottasko pois jos kommentti on tyhjä, kun sillon muuten noi tekstit ei oo keskellä ku tää ottaa siitä pienen siivun

```suggestion
            {review.comment?.length > 0 ? (
              <p className="mt-2 text-stone-900 dark:text-stone-100">
                {review.comment}
              </p>
            ) : null}
```